libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/evcs/client/EvcsClient.java

```diff
@@ -190,8 +192,38 @@ public void updateUserVCs(String userId, List<EvcsUpdateUserVCsDto> evcsUserVCsT
         }
     }
 
-    private URI getIdentityUri() throws URISyntaxException {
-        return getUri(IDENTITY_SUB_PATH, null, null);
+    public void invalidateStoredIdentityRecord(String userId) throws EvcsServiceException {
+        LOGGER.info(
+                LogHelper.buildLogMessage("Preparing to invalidate user's stored identity record"));
+        try {
+            HttpRequest.Builder httpRequestBuilder =
+                    HttpRequest.newBuilder()
+                            .uri(getIdentityUri("invalidate"))
+                            .POST(
+                                    HttpRequest.BodyPublishers.ofString(
+                                            OBJECT_MAPPER.writeValueAsString(
+                                                    new EvcsInvalidateStoredIdentityDto(userId))))
+                            .header(
+                                    X_API_KEY_HEADER,
+                                    configService.getSecret(ConfigurationVariable.EVCS_API_KEY))
+                            .header(CONTENT_TYPE, ContentType.APPLICATION_JSON.toString());
+
+            sendHttpRequest(httpRequestBuilder.build());
+        } catch (URISyntaxException e) {
+            throw new EvcsServiceException(
+                    HTTPResponse.SC_SERVER_ERROR, ErrorResponse.FAILED_TO_CONSTRUCT_EVCS_URI);
+        } catch (JsonProcessingException e) {
+            throw new EvcsServiceException(
+                    HTTPResponse.SC_SERVER_ERROR, ErrorResponse.FAILED_TO_PARSE_EVCS_REQUEST_BODY);
+        }
+    }
+
+    private URI getIdentityUri(String identitySubPaths) throws URISyntaxException {
```

Comment:
```suggestion
    private URI getIdentityUri(String path) throws URISyntaxException {
```
(V minor) It's sub-IDENTITY_SUB_PATH and the plural is slightly misleading, so might just be better to call it path as it's implied it's getting /identity at least from the method name